src/main/java/com/dataloom/search/SearchApi.java

```diff
@@ -15,13 +15,37 @@
 	String KEYWORD = "kw";
 	String ENTITY_TYPE_ID = "eid";
 	String PROPERTY_TYPE_ID = "pid";
-		
+	
+	/** 
+	 * The query, entityType, and propertyTypes params are all optional,
+	 * but at least one must be specified otherwise an error will be thrown.
+	 * All specified params are required to be present in each entity set returned.
+	 * If entityType and propertyTypes are both specified, the propertyTypes param
+	 * will be ignored.
+	 * 
+	 * @param query
+	 * @param entityType
+	 * @param propertyTypes
+	 * @return JSON string of matching entity set metadata
+	 */
 	@GET( SEARCH )
 	String executeQueryJson(
 			@Query(KEYWORD) String query,
 			@Query(ENTITY_TYPE_ID) UUID entityType,
 			@Query(PROPERTY_TYPE_ID) Set<UUID> propertyTypes );
 	
+	/** 
+	 * The query, entityType, and propertyTypes params are all optional,
+	 * but at least one must be specified otherwise an error will be thrown.
+	 * All specified params are required to be present in each entity set returned.
+	 * If entityType and propertyTypes are both specified, the propertyTypes param
+	 * will be ignored.
+	 * 
+	 * @param query
+	 * @param entityType
```

Comment:
An optional parameter specifying the entity type of the entity sets that will be returned.